src/index.js

```diff
@@ -105,6 +114,9 @@ export default class VueI18n {
     })
   }
 
+  _checkLocaleMessage (locale: Locale, level: AllowHtmlFormattingLevel, message: LocaleMessageObject): void {
+  }
```

Comment:
I think it would be helpful if error message suggest users to read [Formatting](https://kazupon.github.io/vue-i18n/guide/formatting.html#html-formatting) and [use component interpolation](https://kazupon.github.io/vue-i18n/guide/interpolation.html#basic-usage)